xgi/drawing/draw.py

```diff
@@ -755,22 +779,65 @@ def _color_arg_to_dict(arg, ids, cmap):
     ------
     TypeError
         If a string, dict, iterable, or NodeStat/EdgeStat is not passed.
+
+    Notes
+    -----
+    For the iterable of values, we do not accept numpy arrays or tuples,
+    because there is the potential for ambiguity.
```

Comment:
Basically the ambiguity is that a color can be specified as a tuple of floats (R,G,B) or (R,G,B,alpha) and if you had a hypergraph with 3 or 4 nodes, it would break if you also accepted tuples of floats in the case where we map float values to a color map.